src/webapp/src/auth/AuthenticationOptions.tsx

```diff
@@ -1,7 +1,68 @@
+import { Button } from '@small-ads/ui';
 import React from 'react';
+import styled from 'styled-components';
+import { FacebookSquare } from '@styled-icons/boxicons-logos';
+
+const LoginScreen = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: calc(100vh - 64px);
+`;
+
+const LoginMain = styled.div`
+  align-self: center;
+  flex: 1 1;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const LoginSign = styled.div`
+  font-size: 3rem;
+  letter-spacing: -0.066875rem;
+  font-weight: 700;
+`;
+
+const LoginForm = styled.div`
```

Comment:
I dont see any `form` per se. Form is usually html elements. Will end up being a little bit confusing when we actually start having forms.

Reply:
I copied from Vercel :(. I will start renaming their things when they don't make too much sense, thanks!

Reply:
Sorry for being special about those things. But hope you see my logic 👍